$MirOS: ports/devel/libffi/patches/patch-include_ffi_common_h,v 1.1.1.1 2008/08/08 15:53:57 tg Exp $
--- include/ffi_common.h.orig	2008-07-12 07:43:00.000000000 +0200
+++ include/ffi_common.h	2009-10-11 14:11:58.000000000 +0200
@@ -62,6 +62,7 @@ void ffi_type_test(ffi_type *a, char *fi
 #define FFI_ASSERT_VALID_TYPE(x)
 #endif
 
+#undef ALIGN
 #define ALIGN(v, a)  (((((size_t) (v))-1) | ((a)-1))+1)
 #define ALIGN_DOWN(v, a) (((size_t) (v)) & -a)
 
